### PR TITLE
Nerf +Insist

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by ghost** [comment]
Merge your PR's

**Comment by DeathlyPlays** [comment]
Meant to be separate

**Comment by DeathlyPlays** [comment]
Lemme fix this small thing

File: mods/sgssb/moves.js [hunk 1]
```diff
@@ -617,6 +617,7 @@ exports.BattleMovedex = {
 		priority: 1,
 		self: {
 			boosts: {
+				chance: 30,
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js]: Needs to be tested because i'm not sure this will work.

**Pull request closed without merging**